routes/api_v1.php

```diff
@@ -1871,6 +1871,10 @@
 
             // 25 requests per minute
             Route::get('{promo_code_val}/apply', ['middleware' => ['rate.limit:25,1'], 'uses' => 'OAuth2SummitPromoCodesApiController@preValidatePromoCode']);
+
+            Route::group(['prefix' => 'all'], function () {
+                Route::put('send', ['middleware' => 'auth.user', 'uses' => 'OAuth2SummitPromoCodesApiController@sendSponsorPromoCodes']);
```

Comment:
this is not correct 
SDS saids
PUT api/v1/summits/{id}/sponsors/all/promo-codes/all/send